### PR TITLE
doc/nrf/../software_maturity: added mcuboot record

### PR DESCRIPTION
**Pull request by nvlsianpu**
Added summary table on MCUboot maturity level.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -1881,6 +1881,59 @@ PSA Crypto support
          - Supported
          - Supported
 
+
+MCUboot bootloader
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:1885]: The MCUboot bootloader shouldn't be a section under Security Feature Support. This should be a separate new section.

**Comment by divipillai** [comment]
The backport label could be removed as the 2.8 SW maturity level page is already published?

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 2]
```diff
@@ -1881,6 +1881,59 @@ PSA Crypto support
...
+         - nRF52833
+         - nRF52840
+         - nRF5340
+         - nRF54H20
+         - nRF54L15
+         - nRF9131
+         - nRF9151
+         - nRF9160
+         - nRF9161
+       * - **Immutable Bootloader as part of build**
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/software_maturity.rst:1908]: Doesn't the first row overlap with what's under the NSIB section (though the column for nRF54L15 isn't updated in the old content)?

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -1881,6 +1881,59 @@ PSA Crypto support
          - Supported
          - Supported
 
+
+MCUboot bootloader
+==================
+
+.. toggle::
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/software_maturity.rst:1888]: Should be moved above or below Security Features support as a separate section, as already suggested by Divya

```
MCUboot bootloader
******************
```

File: doc/nrf/releases_and_maturity/software_maturity.rst (hunk 2 above)
Comment (nvlsianpu) [reply doc/nrf/releases_and_maturity/software_maturity.rst:1908]: MCUboot might be instanced as immutable or updatable. Bootloader->MCUboot

**Review by hellesvik-nordic — APPROVED** [review]
I looked at the docs preview, and it looks as I would expect

**Review by annwoj — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 4]
```diff
@@ -1959,6 +1959,58 @@ Trusted storage implements the PSA Certified Secure Storage APIs without TF-M.
         - Supported
         - Supported
 
+MCUboot bootloader
+******************
+
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:1964]: Need to add an intro sentence here :

_The following table indicates the software maturity levels of the support for each MCUboot bootloader feature:_

**Review by divipillai — APPROVED** [review]
Approved with a minor suggestion

**Pull request merged**